cmd/list.go

```diff
@@ -18,93 +18,119 @@ package cmd
 import (
 	"encoding/json"
 	"fmt"
-	"net/http"
+	// "net/http"
 	"os"
 	"strings"
 	"text/tabwriter"
 
 	"github.com/spf13/cobra"
 )
-
-// CollStruct : Each collection contains following information to be displayed
-type CollStruct struct {
+// The list command gets a set of "stacks" back from the CLI Service.  
+// There are 2 distinct structs for the returned stacks:
+//   - KabStruct represents the JSON returned for the Kabanero stacks.  
```

Comment:
Because the json we are receiving now is more complicated and nested, can you put something like an indented example of what the structure is that is being sent with these new structs, its a little unclear of what might be nested in the other in a current request